Day-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Header.css';
+
+const ImageThumbnail = () => {
+    return (
```

Comment:
Look into modifying SVG when you want to see a different way of handling images.